### PR TITLE
Follow-up on routing commuting 2q gates

### PR DESCRIPTION
**Pull request by eggerdj**
### Summary

This is a PR to address the remaining comments in #7813 .

### Details and comments

See the response to the comments in #7813.

### CODE REVIEW

File: qiskit/transpiler/passes/routing/commuting_2q_gate_routing/commuting_2q_block.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def __init__(self, node_block: Iterable[DAGOpNode]) -> None:
             )
 
         super().__init__(
-            "commuting_2q_block", num_qubits=len(qubits), params=[], label="Commuting 2q gates"
+            "commuting_2q_blocks", num_qubits=len(qubits), params=[], label="Commuting 2q gates"
```
Comment (1ucian0) [inline qiskit/transpiler/passes/routing/commuting_2q_gate_routing/commuting_2q_block.py:51]: This comes from https://github.com/Qiskit/qiskit-terra/pull/7813#discussion_r856217819

I actually think the name of the class should be the one changing. Is a single block that contains 2q gates that commute.

Comment (eggerdj) [reply qiskit/transpiler/passes/routing/commuting_2q_gate_routing/commuting_2q_block.py]: Done here: [401b07c](https://github.com/Qiskit/qiskit-terra/pull/7979/commits/401b07c6ea0283f3ebb0f6ca71a3ff1607271470)

File: qiskit/transpiler/passes/routing/__init__.py [hunk 2]
```diff
@@ -18,5 +18,5 @@
 from .stochastic_swap import StochasticSwap
 from .sabre_swap import SabreSwap
 from .bip_mapping import BIPMapping
-from .swap_strategies.commuting_2q_gate_router import Commuting2qGateRouter
-from .swap_strategies.swap_strategy import SwapStrategy
+from .commuting_2q_gate_routing.commuting_2q_gate_router import Commuting2qGateRouter
+from .commuting_2q_gate_routing.swap_strategy import SwapStrategy
```
Comment (ajavadia) [inline qiskit/transpiler/passes/routing/__init__.py:22]: can you remove SwapStrategy from init? It's not a pass (contrary to all the other things in this namespace). I think people can import it from its own directory.

**Review by ajavadia — COMMENTED** [review]
looks good, thanks for doing this follow up

**Review by ajavadia — APPROVED** [review]

**Pull request merged**